### DISCUSSION TITLE
Technical Onboarding Updates

### DISCUSSION THREAD

**Discussion by tessawiedmann** [microsoft/SATechnicalOnboarding#27] [Announcements]
Hey Student Ambassador Community!

The Technical Onboarding repo was recently updated on April 18. The update included replacing the old sections 1-4 which included creating a repository, cloning it, creating a static web app, and customizing it, with a new section 1 which now points to a [new template repo ](https://github.com/education/codespaces-project-template-js) created by our awesome Cloud Advocate team where you still do the same steps and create your own Portfolio site, but now setup is even easier through the use of GitHub Codespaces! 

This [new template repository ](https://github.com/education/codespaces-project-template-js)shows the power of GitHub Codespaces and GitHub Copilot and hopefully will make the Ambassador Technical Onboarding experience even smoother and more powerful by introducing you to more tools. If you have already completed Technical Onboarding, feel free to try the[ new template repo ](https://github.com/education/codespaces-project-template-js)out yourself!

**Comment by Harshithanunnaguppala** [comment]
Am getting "RepositoryToken is invalid. Provided token has invalid permissions and cannot be used to setup Github Action CI/CD. Admin rights are required for the repository." What to do?

**Reply by dantrixxheron** [reply]
I got the same problem when I tried to create the static web app, maybe it's not the right way but I created the app from Azure page by connecting my GitHub account and it was created successfully, only it won't show up in codespaces. If there is another way to do it, I would also like to know it!
This guy shows how to do it: https://github.com/microsoft/SATechnicalOnboarding/discussions/20#discussioncomment-5755127

**Reply by Harshithanunnaguppala** [reply]
Thank you!

**Comment by Akintunde76** [comment]
I am struggling to get my codespace to work successfully, can anyone help me out please  https://github.com/Akintunde76/BenjaminAkinmoyeje-/issues/1

**Reply by ManuSquall** [reply]
Hello @Akintunde76 , did you make it work since?

**Reply by Akintunde76** [reply]
No I did not make it work, I couldn't get my codespace working - please help. It tells me my codespace is 100%

**Comment by Sunilshah-7** [comment]
I am struggling to setup my Azure account, it says to subscribe to a plan to host in Azure platform. Do I have to subscribe even from ambassador student account (xxx.xxx@studentambassdors.com) ?
